modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/package.json

```diff
@@ -35,7 +35,6 @@
 			"Evaluator",
 			"FormBuilder",
 			"FormRenderer",
-			"LayoutProvider/handlers/fieldDeletedHandler",
```

Comment:
@alinedoleron, you changed the content of this commit, however this change wasn't replicated to the commit message, can you please update the commit message?